content/en/blog/_posts/2019-07-09-some-apis-are-being-deprecated.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: blog
+title: 'APIs Deprecating Soon: Here\'s What You Need to Know'
+date: 2019-07-08
+---
+
+**Author**: Vallery Lancey (Lyft)
+
+As the Kubernetes API evolves, APIs are periodically reorganized or upgraded.
+When APIs evolve, the old API is deprecated and eventually removed.
+
+The 1.16 release will deprecate APIs for four services:
+
+* NetworkPolicy
+* PodSecurityPolicy
+* DaemonSet, Deployment, StatefulSet, and ReplicaSet
+* Ingress
+
+None of these resources will be removed from Kubernetes or deprecated in any way.
+However, to continue using these resources, you must use a current version of
+the Kubernetes API.
+
+# Migration Details
+
+* NetworkPolicy: will no longer be served from **extensions/v1beta1** in **v1.16**.
+  * Migrate to the networking.k8s.io/v1 API, available since v1.8. Existing persisted
+    data can be retrieved/updated via the networking.k8s.io/v1 API.
+* PodSecurityPolicy: will no longer be served from **extensions/v1beta1** in **v1.16**.
+  * Migrate to the policy/v1beta1 API, available since v1.10. Existing persisted
+    data can be retrieved/updated via the policy/v1beta1 API.
+* DaemonSet, Deployment, StatefulSet, and ReplicaSet:  will no longer be served
+from **extensions/v1beta1**, **apps/v1beta1**, or **apps/v1beta2** in **v1.16**.
+  * Migrate to the apps/v1 API, available since v1.9. Existing persisted data
+    can be retrieved/updated via the apps/v1 API.
+* Ingress: will no longer be served from **extensions/v1beta1** in **v1.18**.
+  * Migrate to the networking.k8s.io/v1beta1 API. Existing persisted data can be
+    retrieved/updated via the networking.k8s.io/v1beta1 API.
+
+# What To Do
+
+Kubernetes 1.16 is due to be released in September 2019, so be sure to audit
+your configuration and integrations now!
+
+* Change YAML files to reference the newer APIs
+* Update custom integrations and controllers to call the newer APIs
+* Update third party tools (ingress controllers, continuous delivery systems)
+to call the newer APIs
+
+Migrating to the new Ingress API will only require changing the API path - the
+API fields remain the same. However, migrating other resources (EG Deployments)
+will require some updates based on changed fields. You can use the
+`kubectl convert` command to automatically convert an existing object:
+`kubectl convert -f <file> --output-version <API>/<version>`
```

Comment:
nit: `--output-version <group>/<version>`

might be helpful to give an example, e.g. `kubectl convert -f extensions-v1beta1-deployment.yaml --output-version apps/v1 > apps-v1-deployment.yaml`